tools/Azure.Mcp.Tools.KeyVault/src/Commands/Admin/AdminSettingsGetCommand.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Mcp.Core.Commands;
+using Azure.Mcp.Core.Commands.Subscription;
+using Azure.Mcp.Tools.KeyVault.Options;
+using Azure.Mcp.Tools.KeyVault.Services;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Mcp.Tools.KeyVault.Commands.Admin;
+
+public sealed class AdminSettingsGetCommand(ILogger<AdminSettingsGetCommand> logger) : SubscriptionCommand<BaseKeyVaultOptions>
+{
+    private const string CommandTitle = "Get Key Vault Administration Settings";
```

Comment:
Maybe we should use:
```suggestion
    private const string CommandTitle = "List Key Vault Account Settings";
```